app/graphql/mutations/create_election_mutation.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Mutations
+  class CreateElectionMutation < GraphQL::Schema::Mutation
+    argument :title, String, required: true
+    argument :data, String, required: true
+
+    field :outcome, String, null: true
+    def resolve(title:, data:)
+      data_hash = Digest::SHA256.hexdigest(data)
+      election_form = ElectionForm.new(title: title, status: "Published", client: Client.last,
+                                       data: data,
+                                       data_hash: data_hash, log_type: "createElection")
+
+      if CreateElection.call(election_form)
+        { outcome: "Success!" }
```

Comment:
Here I'd follow [the graphql gem convention for the error raising](https://graphql-ruby.org/mutations/mutation_errors). Also, if things go ok, I'd return just the created election (it should be broadcasted by the `CreateElection` to allow you to return something like `{ election: election }`.
 
You can find an example of how to do this in the decidim app: https://github.com/decidim/decidim/blob/develop/decidim-comments/app/resolvers/decidim/comments/vote_comment_resolver.rb#L18

Finally, what do you think about using resolver classes to implement all the mutation logic?
I'm not sure, but I understand that it consists only on a regular class with an initializer and a `call` method.

Reply:
I tried the `Resolver` but I found out that the `Mutation` is a specialized subclass of `Resolver`, I was seeing that things were getting repetitive when adding a new `Resolver` class. Here is a little more info:
https://github.com/rmosolgo/graphql-ruby/blob/master/guides/fields/resolvers.md

Tell me what you think :)